### PR TITLE
chore: upgrade to Go 1.18.7 (1.9)

### PR DESCRIPTION
**Pull request by bnpfeife**

### CODE REVIEW

File: tests/backup_restore_test.go [hunk 1]
```diff
@@ -22,15 +21,18 @@ func TestServer_BackupAndRestore(t *testing.T) {
...
-	partialBackupDir, _ := ioutil.TempDir("", "backup")
+	partialBackupDir, _ := os.MkdirTemp("", "backup")
 	defer os.RemoveAll(partialBackupDir)
 
-	portableBackupDir, _ := ioutil.TempDir("", "backup")
+	portableBackupDir, _ := os.MkdirTemp("", "backup")
 	defer os.RemoveAll(portableBackupDir)
 
+	shardBackupDir, _ := os.MkdirTemp("", "backup")
+	defer os.RemoveAll(shardBackupDir)
```
Comment (jeffreyssmith2nd) [inline tests/backup_restore_test.go:34]: Why were these lines introduced?

**Comment by jdstrand** [comment]
@bnpfeife - what is the status of this?

File: tests/backup_restore_test.go (hunk 1 above)
Comment (bnpfeife) [reply tests/backup_restore_test.go]: Oh, I missed this comment. Oh, it looks like I accidentally added this when I cherry-picked the commit. I've removed these lines in a new commit.

**Comment by jdstrand** [comment]
@jeffreyssmith2nd - it looks like @bnpfeife addressed your question. Re-requesting your review.

**Comment by jdstrand** [comment]
> @jeffreyssmith2nd - actually, hold on.

@jeffreyssmith2nd - this is ready now.

**Review by jeffreyssmith2nd — APPROVED** [review]

**Pull request merged**